internal/workflow.go

```diff
@@ -1565,11 +1565,97 @@ const DefaultVersion Version = -1
 // CadenceChangeVersion is used as search attributes key to find workflows with specific change version.
 const CadenceChangeVersion = "CadenceChangeVersion"
 
+// GetVersionOption configures GetVersion behavior
+type GetVersionOption func(*GetVersionOptions)
+
+// GetVersionOptions contains options for GetVersion
+type GetVersionOptions struct {
```

Comment:
I understand this is not supposed to be used directly and instead helper functions below are recommended. However this is still public and open for misuse. Let's clarify the precedence of the fields in the comments

Reply:
it's in an internal folder, so this isn't visible outside the client's library (unless we alias it out, which isn't happening below)